crates/uv-resolver/src/lock/export/mod.rs

```diff
@@ -73,8 +74,12 @@ impl<'lock> ExportableRequirements<'lock> {
             let dist = target
                 .lock()
                 .find_by_name(root_name)
-                .expect("found too many packages matching root")
-                .expect("could not find root");
+                .map_err(|_| LockErrorKind::MultipleRootPackages {
+                    name: root_name.clone(),
+                })?
+                .ok_or_else(|| LockErrorKind::MissingRootPackage {
+                    name: root_name.clone(),
+                })?;
```

Comment:
We now catch this early, but there's no reason to panic with there's different paths to accessing this code.